packages/scaffold-config/src/ct-detect-third-party.ts

```diff
@@ -25,6 +26,46 @@ const thirdPartyDefinitionPrefixes = {
   globalPrefix: 'cypress-ct-',
 }
 
+const ROOT_PATHS = [
```

Comment:
Nit... should this be `ROOT_FILES`?

Reply:
I used the term paths because some of them are directories and some are files, for example `.git` is a directory.